### PR TITLE
documentation-github-flow-update

### PR DESCRIPTION
**Pull request by 13Homer13**
Updated flow and screenshots for github documentation.

### CODE REVIEW

File: site/source/developer-docs/documentation.rst [hunk 1]
```diff
@@ -9,100 +9,103 @@ If you would like to contribute to the documentation please follow the instructi
...
-        #. Describe your changes in the first field, click "Create new branch for this commit and start a pull request," name the branch in the second field and click "Propose changes"
+        #. Tap "documentation".
 
-            .. figure:: /_static/images/contribution/propose-change.png
-                :width: 60%
+               .. figure:: /_static/images/contribution/documentation-update-step4.png
+                :width: 70%     
 
-        #. Click "documentation":
+        #. Tap "new pull request" by the side of the branch you just created:
```
Comment (kn0wmad) [inline site/source/developer-docs/documentation.rst:51]:
```suggestion
        #. Click "Compare & pull request" in the yellow box (if this does not appear, go to "Pull Requests -> New"):
```

File: site/source/developer-docs/documentation.rst [hunk 2]
```diff
@@ -9,100 +9,103 @@ If you would like to contribute to the documentation please follow the instructi
...
-        #. Click "new pull request" by the side of the branch you just created:
+        #. Now ensure that the base repository is Start9Labs/documentation, then tap "Create pull request".
 
-               .. figure:: /_static/images/contribution/documentation-then-new-pr.png
-                :width: 60%
+            .. figure:: /_static/images/contribution/documentation-update-step6.png
+                :width: 70%
 
-        #. Now ensure that the base repository is Start9Labs/documentation, then click the cog on the right and select someone from the Start9 team to review your PR then finally click "Create pull request".
+        #. Once reviewed we will either suggest changes or approve it if no changes are necessary. If we reject your PR we will try to offer an explanation of why the PR was not needed.
```
Comment (kn0wmad) [inline site/source/developer-docs/documentation.rst:61]:
```suggestion
        #. From here, a Start9 team member will review your PR, either suggesting changes or approving it if no changes are necessary.
```

File: site/source/developer-docs/documentation.rst [hunk 3]
```diff
@@ -9,100 +9,103 @@ If you would like to contribute to the documentation please follow the instructi
...
 
-        #. If you are unable to request review you can still click "Create pull request" - once reviewed we will either suggest changes or approve it if no changes are necessary. If we reject your PR we will try to offer an explanation of why the PR was not needed.
+        1. Tap "Add file" and then "Create new file".
 
-        Thank you for contributing to Start9's documentation!
+            .. figure:: /_static/images/contribution/documentation-new-step1.png
+                :width: 70%
 
-    .. group-tab:: Creating a new page:
+        2. Name and enter the contents of your first article and then tap "Commit changes".
```
Comment (kn0wmad) [inline site/source/developer-docs/documentation.rst:72]:
```suggestion
        2. Name and enter the contents of your first article and then click "Commit changes".  Remember to put this in the correct location in the file structure.
```

File: site/source/developer-docs/documentation.rst [hunk 4]
```diff
@@ -9,100 +9,103 @@ If you would like to contribute to the documentation please follow the instructi
...
 
-        #. Click "Add file" and then "Create new file":
+            .. figure:: /_static/images/contribution/documentation-new-step2.png
+                :width: 70%
 
-            .. figure:: /_static/images/contribution/new-file.png
-                :width: 60%
+        3. Describe your article in the first field, tap "Create new branch for this commit and start a pull request," name the branch in the second field and tap "Propose new file".
```
Comment (kn0wmad) [inline site/source/developer-docs/documentation.rst:77]:
```suggestion
        3. Describe your article in the first field, click "Create new branch for this commit and start a pull request," name the branch in the second field and click "Propose new file".
```

File: site/source/developer-docs/documentation.rst [hunk 5]
```diff
@@ -9,100 +9,103 @@ If you would like to contribute to the documentation please follow the instructi
...
 
-        #. Name and enter the contents of your first article:
+            .. figure:: /_static/images/contribution/documentation-new-step3.png
+                :width: 70%
 
-            .. figure:: /_static/images/contribution/name-write-first-article.png
-                :width: 60%
+        4. Now tap "documentation" and you will likely see this "Compare and pull request" button - if so tap it and head to the next step.
```
Comment (kn0wmad) [inline site/source/developer-docs/documentation.rst:82]:
```suggestion
        4. Now click "documentation" and then click the "Compare and pull request" button.  If this option does not appear, see the note below.
```

File: site/source/developer-docs/documentation.rst [hunk 6]
```diff
@@ -9,100 +9,103 @@ If you would like to contribute to the documentation please follow the instructi
...
-            .. figure:: /_static/images/contribution/propose-new-file.png
-                :width: 60%
 
-        #. Now click "documentation" and you will likely see this "Compare and pull request" button - if so click it and head to the next step:
+        
+        .. note::
 
-            .. figure:: /_static/images/contribution/if-orange-field.png
-                :width: 60%
+            If you don't see "Compare & pull request" button, then tap "branches" instead, then tap "New pull request" next to the branch you created.
```
Comment (kn0wmad) [inline site/source/developer-docs/documentation.rst:94]:
```suggestion
            If you don't see the "Compare & pull request" button, click "branches", then click "New pull request" next to the branch you created.
```

File: site/source/developer-docs/documentation.rst [hunk 7]
```diff
@@ -9,100 +9,103 @@ If you would like to contribute to the documentation please follow the instructi
...
-            .. figure:: /_static/images/contribution/branches-instead.png
-                :width: 60%
+                .. figure:: /_static/images/contribution/documentation-new-step5.2.png
+                    :width: 60%
 
-            .. figure:: /_static/images/contribution/branches-then-new-pr.png
-                :width: 60%
 
-        #. Now ensure that the base repository is Start9Labs/documentation, then click the cog on the right and select someone from the Start9 team to review your PR then finally click "Create pull request".
+        5. Now ensure that the base repository is Start9Labs/documentation, then finally tap "Create pull request".
```
Comment (kn0wmad) [inline site/source/developer-docs/documentation.rst:104]:
```suggestion
        5. Ensure that the base repository is Start9Labs/documentation, then click "Create pull request".
```

File: site/source/developer-docs/documentation.rst [hunk 8]
```diff
@@ -9,100 +9,103 @@ If you would like to contribute to the documentation please follow the instructi
...
 
-            .. figure:: /_static/images/contribution/cog-create-pr.png
-                :width: 60%
+            .. figure:: /_static/images/contribution/documentation-new-step6.png
+                :width: 70%
 
-        #. If you are unable to request review you can still click "Create pull request" - once reviewed we will either suggest changes or approve it if no changes are necessary. If we reject your PR we will try to offer an explanation of why the PR was not needed.
+        6. Once reviewed we will either suggest changes or approve it if no changes are necessary. If we reject your PR we will try to offer an explanation of why the PR was not needed.
```
Comment (kn0wmad) [inline site/source/developer-docs/documentation.rst:109]:
```suggestion
        6. From here, a Start9 team member will review your PR, either suggesting changes or approving it if no changes are necessary.
```

**Review by kn0wmad — CHANGES_REQUESTED** [review]

**Comment by kn0wmad** [comment]
Closes #449

File: site/source/developer-docs/documentation.rst [hunk 9]
```diff
@@ -9,100 +9,103 @@ If you would like to contribute to the documentation please follow the instructi
...
 
-    .. figure:: /_static/images/contribution/github-signup.png
-        :width: 60%
+    .. figure:: /_static/images/contribution/documentation-fork-step2.png
+        :width: 70%
 
 #. Head to `Start9's documentation repo. <https://github.com/start9labs/documentation>`_
 
-#. Fork the repository by clicking here:
+#. Fork the repository by taping here:
```
[the remark this answers is not in the document]
Comment (13Homer13) [reply site/source/developer-docs/documentation.rst:17]: Yeah I will keep in mind.

**Review by kn0wmad — APPROVED** [review]

**Pull request merged**